alternator/server.cc

```diff
@@ -374,6 +378,7 @@ future<std::string> server::verify_signature(const request& req, const chunked_c
         } catch (const api_error& e) {
             _executor._stats.authentication_failures++;
             if (_enforce_authorization.get() == db::tri_mode_restriction_t::mode::WARN) {
```

Comment:
If I understand correctly, you don't print to the log when there's actually an error.
Don't hold the merge because of it. But if you have a new version, I suggest adding it.

Logs are a good way to show the history of what happened.

Reply:
Added this feature (logging even if when an errror would be generated) to the new version. Basically the new version (explained below) allows you to set `alternator_enforce_authorization=true` to generate errors and also set `alternator_warn_authorization=true` to additionally add WARN-level log messages.